ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -12,10 +12,18 @@ nginx_sites:
       hosts: formplayer
       port: "{{ formplayer_port }}"
       method: "hash $cookie_sessionid consistent"
+   local_ips:
+    - '10.247.24.0/24'
    mappings:
     - name: "$digest_submission"
-      variable: "$http_authorization"
-      value: "$server_name"
+      variable: '"$is_remote:$http_authorization"'
+      mappings:
+        - key: '"~^1:"'
+          value: "$server_name"
+        - key: '"~^1:.+"'
```

Comment:
Is this mapping needed? Since it has the same results as the default I think you can leave it out.